### PR TITLE
Styled Navbar

### PR DESCRIPTION
**Pull request by hariom1625**
### Solves Issue #16 
Description: Styled the Navbar as proposed in the wireframe.
PS: The cover image is being added only for illustration purpose.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

### CODE REVIEW

**Comment by hariom1625** [comment]
> ### Solves Issue #16
> Description: Styled the Navbar as proposed in the wireframe.
> 
> #### Checklist
> * [x]  I've followed the Contributing guidelines provided in the repository.
> * [x]  I've made the changes which were demanded in the linked issue.
> * [x]  I've tested my code on a Chromium based browser.
> * [x]  I've tested my code on Mozilla Firefox.
> * [x]  My code gave a clean console on debugging. (no warnings/errors)

File: client/src/components/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.lander-img{
+width:auto;
+height: auto;
```
Comment (EmperorYP7) [inline client/src/components/Home/Home.css:3]: Do take care of the indentation.

File: client/src/components/Home/Home.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './Home.css';
-
+import shadow from './blue_blur.png';
 function Home() {
   return (
     <div className="home-component">
-      <h1>This is the Home page.</h1>
+      <img className="lander-img img-fluid" src={shadow} alt="lander"/>
```
Comment (EmperorYP7) [inline client/src/components/Home/Home.js:8]: Using `react-lazyload` would help (I guess) as suggested by @ankiiitraj

File: client/src/components/Navbar/Navbar.css [hunk 3]
```diff
@@ -1,4 +1,133 @@
...
+}
+
+.search-bar{
+      border-radius:35px;
+      width:180px!important;
+      height: 30px;
+      margin-right: 25px;
+      margin-left: 20px;
+      padding-left: 35px;
+      }
```
Comment (EmperorYP7) [inline client/src/components/Navbar/Navbar.css:48]: Stick the closing parenthesis to the margin here.

File: client/src/components/Navbar/Navbar.css [hunk 4]
```diff
@@ -1,4 +1,133 @@
...
+
+.search-icon{
+
+      position:absolute;
+      opacity: 0.6;
+      right: 190px;
+      bottom: 19px;
+      height: 19px;
+      }
```
Comment (EmperorYP7) [inline client/src/components/Navbar/Navbar.css:57]: Here as well

File: client/src/components/Navbar/Navbar.css [hunk 5]
```diff
@@ -1,4 +1,133 @@
...
+      .search-icon{
+
+            position:absolute;
+            opacity: 0.6;
+            right: 310px;
+            bottom: 30px;
+            height: 20px;
+      }
+
+}
```
Comment (EmperorYP7) [inline client/src/components/Navbar/Navbar.css:107]: Do correct the indentation of parenthesis in the media querries. It should be clear where the querry is ending.

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
This PR will be merged only after #20 is merged. Avoid changing `Navbar.js` to minimize conflicts later. The build is looking beautiful.. Good job @hariom1625 ! 🤩

File: client/src/components/Navbar/Navbar.css (hunk 3 above)
Comment (hariom1625) [reply client/src/components/Navbar/Navbar.css]: Done

File: client/src/components/Navbar/Navbar.css (hunk 4 above)
Comment (hariom1625) [reply client/src/components/Navbar/Navbar.css]: Done 😅

File: client/src/components/Home/Home.css (hunk 1 above)
Comment (hariom1625) [reply client/src/components/Home/Home.css:3]: As I postscript 😅 the cover image was for illustration purpose so I didn't decor that code as I haven't started working on that.

File: client/src/components/Home/Home.js (hunk 2 above)
Comment (hariom1625) [reply client/src/components/Home/Home.js:8]: Will make use of it from next time, it was only for visualizing the navbar with a background other than white :-)

File: client/src/components/Navbar/Navbar.css (hunk 5 above)
Comment (hariom1625) [reply client/src/components/Navbar/Navbar.css:105]: Fixed.

**Review by ankiiitraj — APPROVED** [review]

**Review by EmperorYP7 — APPROVED** [review]
Good to go! 🥳  Waiting for #20 to get merged. Do change the title for this PR for avoiding confusion.

**Comment by ankiiitraj** [comment]
some conflicts and you're ready to go

**Comment by hariom1625** [comment]
> some conflicts and you're ready to go

I have resolved the conflicts you can review the changes now :-)

**Review by EmperorYP7 — APPROVED** [review]
LGTM! 🚀

**Review by ankiiitraj — APPROVED** [review]

**Comment by ankiiitraj** [comment]
nice work @hariom1625

**Comment by hariom1625** [comment]
> nice work @hariom1625

Thank you sir.

**Pull request merged**